docs/metrics.md

```diff
@@ -0,0 +1,14 @@
+# Metrics
+
+Prediction objects have a `metrics` field. This normally includes `predict_time` and `total_time`. Official language models have metrics like `input_token_count`, `output_token_count`, `tokens_per_second`, and `time_to_first_token`. Currently, custom metrics from Cog are ignored when running on Replicate. Official Replicate-published models are the only exception to this. When running outside of Cog, you can emit custom metrics like this:
```

Comment:
What does "When running outside of Cog" mean? Should this be "outside of Replicate"?

Reply:
fixed; we'll also have another chance to review this when merging into main